packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
+icon: ""
+apiversion: v2
+condition: ""
+tags: ""
+appversion: 1.16.0
+deprecated: false
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Alerting Drivers
+  catalog.cattle.io/experimental: "true"
```

Comment:
Is this experimental ? I don't think it should be

Reply:
the annotation is remvoed.